extensions/vscode/vscode.wake

```diff
@@ -57,54 +26,55 @@ export def vscode _: Result Path Error =
+        # npm expects to be able to do "which node" so we need to add the entire bin dir to env
+        | editPlanEnvironment (addEnvironmentPath nodeBinDir)
+        # npm writes ./node_modules, cwd to @here so they are in the correct location
         | setPlanDirectory @here
-        # NPM installs its dependencies from the web, and so can't be fully sandboxed.
+        # npm installs its dependencies from the web, and so can't be fully sandboxed.
         | setPlanShare False
         | runJobWith defaultRunner
         | getJobOutputs
 
-    require Pass stdlibSources = sources "share/wake/lib" `.*`
-
-    require Pass stdlib =
-        stdlibSources
-        | findFailFn installRO
-
-    require Pass stdlibJSON = createStdlibJSON stdlibSources
-    require Pass currentDirSources = sources @here `[^/]*`
-    require Pass imagesSources = sources @here `images/.*`
-    require Pass clientSources = sources @here `lsp-client/.*`
-    require Pass serverSources = sources @here `lsp-server/.*`
-    require Pass syntaxSources = sources @here `syntaxes/.*`
+    require Pass images = sources @here `images/.*`
+    require Pass onlineImages = sources @here `online_images/.*`
+    require Pass srcs = sources @here `src/.*`
+    require Pass syntaxes = sources @here `syntaxes/.*`
     require Pass vsceStdin = source "{@here}/vsce-stdin"
+    require Pass tsConfig = source "{@here}/tsconfig.json"
+    require Pass vsCodeIgnore = source "{@here}/.vscodeignore"
+    require Pass license = source "{@here}/LICENSE.txt"
+    require Pass readme = source "{@here}/README.md"
+    require Pass esbuild = source "{@here}/esbuild.js"
+    require Pass languageConfiguration = source "{@here}/language-configuration.json"
 
     def vsceFiles =
+        esbuild,
         vsceStdin,
-        jsFile,
+        tsConfig,
         packageJSON,
-        stdlibJSON,
-        currentDirSources ++
-        imagesSources ++
-        clientSources ++
-        serverSources ++
-        syntaxSources ++
-        stdlib ++
-        nodeModules
+        vsCodeIgnore,
+        license,
+        readme,
+        languageConfiguration,
+        images ++ onlineImages ++ srcs ++ syntaxes ++ nodeModules
 
     require Pass npx =
         whichInEnvPath "npx"
         | getOrFail "Unable to find npx in $PATH".makeError
 
     require Pass outputs =
-        makeExecPlan (npx, "@vscode/vsce@2.15.0", "package", Nil) vsceFiles
+        makeExecPlan (npx, "@vscode/vsce", "package", Nil) vsceFiles
         | setPlanLabel "vsce: Package vsix extension"
-        | editPlanEnvironment (addEnvironmentPath emsdkBinDir)
+        # npx expects to be able to do "which node" so we need to add the entire bin dir to env
+        | editPlanEnvironment (addEnvironmentPath nodeBinDir)
+        # npx looks for ./node_modules so set cwd to @here
         | setPlanDirectory @here
+        # npx asks the user y/n if it should download the program we want to run
```

Comment:
If we turn off `isAtty` does it just hard fail instead of prompting? My question is kind of "can we just delete these two lines and get the same behavior"?

Reply:
It runs fine without `isAtty` but we kept it for two reasons. 1: `npx` will output nice colors 2: IIRC Its the only real test of the feature that we have in the open source repo